velox/exec/tests/TableWriteTest.cpp

```diff
@@ -63,6 +63,8 @@ std::string testModeString(TestMode mode) {
       return "PARTITIONED";
     case TestMode::kBucketed:
       return "BUCKETED";
+    default:
```

Comment:
can we just enumerate all enum values instead of having a default fallback? It's nice to have the compiler complaining about  specific callsites when new enum values are added.

Reply:
I can throw a VELOX_NOT_SUPPORTED exception like Krishna has suggested down below.

Reply:
All modes are enumerated, but gcc still wants a default return so added VELOX_UNSUPPORTED.